src/pages/productCategories/ProductCategories.js

```diff
@@ -0,0 +1,31 @@
+import useRequest from "@ahooksjs/use-request";
+import { Col, Row, Card } from "antd";
+import "./ProductCategories.less";
+
+const ProductCategories = () => {
+    const { data: productCategories } = useRequest("https://4rvey.mocklab.io/api/category");
+
+    return (
+        <div id="productCategory">
+            <div className="container">
+                <Row>
+                    {productCategories && productCategories.map(category => (
+                        <Col>
+                            <Card
+                                hoverable
+                                cover={<img alt={category.name} src={category.image} />}
+                                id="card"
+                            >
+                            <div className="mask">
+                                <p id="title">{category.name}</p>
```

Comment:
change id to class because you cant have id more than 1 in a single page i think. so it will batter if you use class. always use class unless you really know it will be the only 1 on page like header or footer